pets/peer-finder/peer-finder.go

```diff
@@ -40,7 +41,7 @@ var (
 	onStart   = flag.String("on-start", "", "Script to run on start, must accept a new line separated list of peers via stdin.")
 	svc       = flag.String("service", "", "Governing service responsible for the DNS records of the domain this pod is in.")
 	namespace = flag.String("ns", "", "The namespace this pod is running in. If unspecified, the POD_NAMESPACE env var is used.")
-	domain    = flag.String("domain", "cluster.local", "The Cluster Domain which is used by the Cluster.")
+	domain    = flag.String("domain", "", "The Cluster Domain which is used by the Cluster.")
```

Comment:
I think we can come up with a better answer than `cluster.local` (as you see in the lower part of the patch) - the more that we make it easy on the user the better.  I suspect many developers wanting to release their application are not super familiar with their cluster domain setting, they just want their app running as quickly as possible, regardless of what setting some other person set when creating the cluster